mkdocs.yml

```diff
@@ -43,7 +43,7 @@ repo_url: https://github.com/OrchardCMS/OrchardCore
 edit_uri: edit/main/src/
 
 # Options
-docs_dir: src
+docs_dir: src/docs
```

Comment:
This necessitates all the "docs/" removals. Now not the whole `src` folder is considered the docs site, making builds much faster too.